deploy/ui/form.json

```diff
@@ -1547,6 +1589,27 @@
                   "type": "text",
                   "description": "The secret name within the Java keystore used for https encryption. Generated if empty."
                 },
+                {
+                  "label": "Image Context",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.smartRouter.imageContext e.g. rhpam-7, this param is optional for custom image.",
+                  "description": "The image context to use for smart router.",
```

Comment:
what context stands for, namespace or registry? if so we should update it.

Reply:
Was a part of description in the wrong place, updated